src/components/system_admin_card.tsx

```diff
@@ -67,16 +66,56 @@ const SystemAdminCard: FC<SystemAdminCardProps> = ({
     }
   }
 
-  const [anchorEl, setAnchorEl] = React.useState(null);
+  function getSystemsSeverity(systems: System[]) {
+    let status: Color = "success";
```

Comment:
I'm just going to comment that you apparently did the right thing by declaring this to be type `Color`, but that it kind of bums me out anyway. Anyway, carry on.